### ISSUE TITLE
Problem with streaming music to youtube

### ISSUE THREAD

**Issue by sl13** [scivision/PyLivestream#25]
Hello,
I use the following command to stream all music files of a folder:

` sudo python3 FileGlobLivestream.py stream.ini youtube /home/pi/PyLivestream/NCS "*.mp3" -image pic.jpg `

But I get the error:
```
pi@raspberrypi:~/PyLivestream $ sudo python3 FileGlobLivestream.py stream.ini youtube /home/pi/PyLivestream/NCS "*.mp3" -image pic.jpg 
streaming to YouTube Live these files. Be sure list is correct! 

/home/pi/PyLivestream/NCS/m2.mp3
/home/pi/PyLivestream/NCS/m1.mp3
/home/pi/PyLivestream/NCS/m3.mp3
/home/pi/PyLivestream/NCS/m4.mp3
/home/pi/PyLivestream/NCS/m5.mp3
Traceback (most recent call last):
  File "FileGlobLivestream.py", line 45, in <module>
    playonce(flist, p.image)
  File "FileGlobLivestream.py", line 15, in playonce
    PyLivestream.FileIn(p.ini, p.site, f, loop=False, image=image)
  File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 412, in __init__
    stream = Livestream(ini, site, vidsource, image, loop, infn)
  File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 314, in __init__
    self.osparam()
  File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 117, in osparam
    self.res = get_resolution(self.infn)
  File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 77, in get_resolution
    res = (int(res[0].split('=')[-1]), int(res[1].split('=')[-1]))
ValueError: invalid literal for int() with base 10: ''
```

What could be the problem?

PS: I think you should update the main branch of this repository as it will be displayed first to all who visit it on this page.

**Comment by scivision** [comment]
Thank you. I have to find the time to get to a Windows computer, there are a few Windows-specific bugfixes to make. I am not sure when this will be, but when I get the time I will look at this. It's stuff I can't really check from a virtual machine, it needs to be an actual Windows computer.

**Comment by scivision** [comment]
This was resolved by v1.9.0, thanks!

**Comment by leothewolf** [comment]
> Hello, I use the following command to stream all music files of a folder:
> 
> `sudo python3 FileGlobLivestream.py stream.ini youtube /home/pi/PyLivestream/NCS "*.mp3" -image pic.jpg`
> 
> But I get the error:
> 
> ```
> pi@raspberrypi:~/PyLivestream $ sudo python3 FileGlobLivestream.py stream.ini youtube /home/pi/PyLivestream/NCS "*.mp3" -image pic.jpg 
> streaming to YouTube Live these files. Be sure list is correct! 
> 
> /home/pi/PyLivestream/NCS/m2.mp3
> /home/pi/PyLivestream/NCS/m1.mp3
> /home/pi/PyLivestream/NCS/m3.mp3
> /home/pi/PyLivestream/NCS/m4.mp3
> /home/pi/PyLivestream/NCS/m5.mp3
> Traceback (most recent call last):
>   File "FileGlobLivestream.py", line 45, in <module>
>     playonce(flist, p.image)
>   File "FileGlobLivestream.py", line 15, in playonce
>     PyLivestream.FileIn(p.ini, p.site, f, loop=False, image=image)
>   File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 412, in __init__
>     stream = Livestream(ini, site, vidsource, image, loop, infn)
>   File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 314, in __init__
>     self.osparam()
>   File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 117, in osparam
>     self.res = get_resolution(self.infn)
>   File "/home/pi/PyLivestream/PyLivestream/__init__.py", line 77, in get_resolution
>     res = (int(res[0].split('=')[-1]), int(res[1].split('=')[-1]))
> ValueError: invalid literal for int() with base 10: ''
> ```
> 
> What could be the problem?
> 
> PS: I think you should update the main branch of this repository as it will be displayed first to all who visit it on this page.

Hey can you pls help me with my code?
